packages/formation-react/src/components/SortableTable/SortableTable.mdx

```diff
@@ -37,27 +37,28 @@ import SortableTable from '@department-of-veterans-affairs/formation-react/Sorta
     [
       {
         id: 1,
-        name: 'Joe',
+        name: <span>Joe</span>,
         email: 'joe@joe.com',
         message:'Hello',
         rowClass: 'class'
       },
       {
         id: 2,
-        name: 'Bob',
+        name: <span>Bob</span>,
         email: 'bob@bob.com',
         message:'Hola',
         rowClass: 'class'
       },
       {
         id: 3,
-        name: 'Tom',
+        name: <span>Tom</span>,
```

Comment:
This is just showing that JSX is optional, correct?

Reply:
Correct! :)